HACKING.md

```diff
@@ -118,3 +118,57 @@ Assuming `KUBECONFIG` is set, run end-to-end tests:
 ```
 $ make test-e2e
 ```
+
```

Comment:
I think this documentation belongs in in the `/hack` directory.  Just the `make test-pre-release-ossm` command belongs in HACKING.md, with a pointer to the documentation.

Did you and Andrey discuss this?